package.json

```diff
@@ -96,6 +96,7 @@
     "city-timezones": "^1.2.1",
     "eslint": "^8.34.0",
     "lucide-react": "^0.130.0",
+    "react-phone-input-2": "^2.15.1",
```

Comment:
@zomars is this right here in root? should this not be in packages/ or somewhere else?